python/revng/tupletree.py

```diff
@@ -326,3 +326,47 @@ def factory():
         return TypedList(base_class)
 
     return factory
+
+
+def enum_value_to_index(enum_value: Enum):
```

Comment:
This change seems to be a good candidate for its own commit.

Reply:
Split off